### ISSUE TITLE
This device is not activated, please activate it and try to unlock it again

### ISSUE THREAD

**Issue by Egzotyczny12** [offici5l/MiUnlockTool#65]
hi its me again im sorry to typing to you again but i really need help with this error in mi unlock while unlocking bootloader in the internet some guys sayed that i need to type to you my phone is redmi note 10 pro  and he was changed matherboard by warrianty and the warrianty has ended too last year i think you are my only chance to help me with this error
![5868277670259835787](https://github.com/user-attachments/assets/0d5aebb0-d8e6-47fb-b4d4-5c38038bf40b)

**Comment by tiagocasalribeiro** [comment]
That is probably the error 10013. The community tells us to contact support, support tells us to search in the community, where you only find dozens of people with the same issue. I'm on a mi 11 lite 5G NE, by the way, The second I own. On the first one it worked straight forward, about an year ago...

**Comment by Duarte475** [comment]
I'm also facing this issue. I've spoke with an inside person and I was told that this device can't be unlocked. I'm running a Mi 11 Lite 5G (renoir). If you find a solutions for this, please let me know. In the meantime I'm going to try the unlock steps as if this device was running HyperOS.

**Comment by spellydev** [comment]
same error here, did someone find a solution for this?

**Comment by Duarte475** [comment]
unfortunately I can't seem to find the problem. I've been told that this error appears when you do a motherboard swap, but my phone is brand new. So I really don't know where this is going

**Comment by spellydev** [comment]
> unfortunately I can't seem to find the problem. I've been told that this error appears when you do a motherboard swap, but my phone is brand new. So I really don't know where this is going

eh,  i got my motherboard replaced under warranty. what i will try to do is delete all of my xiaomi accounts and restart the unlock process. if it doesnt work im sayin'bye bye to xiaomi

**Comment by raihan131** [comment]
Did any one find any solutions of this ?

**Comment by tiagocasalribeiro** [comment]
For me the solution was to pay for someone to do it, and making a promise to myself to never buy anything from xiaomi again.

**Comment by raihan131** [comment]
So there is no free solutions to unlock it 😭

**Comment by Duarte475** [comment]
> For me the solution was to pay for someone to do it, and making a promise to myself to never buy anything from xiaomi again.

were they able to unlock it ? And was your motherboard the original one ?

**Comment by tiagocasalribeiro** [comment]
> > For me the solution was to pay for someone to do it, and making a promise to myself to never buy anything from xiaomi again.
> 
> were they able to unlock it ? And was your motherboard the original one ?

Yes, and yes. Look for ncunlock(there is a site, which looks sketchy but is legit), talk to him on telegram, he got my phone unlock in 15/20minutes. You don't have to trust me, people on forums also talk about him, thats how I actually learn about him.

**Comment by Berkwe** [comment]
> 

> > > For me the solution was to pay for someone to do it, and making a promise to myself to never buy anything from xiaomi again.
> > 
> > 
> > were they able to unlock it ? And was your motherboard the original one ?
> 
> Yes, and yes. Look for ncunlock(there is a site, which looks sketchy but is legit), talk to him on telegram, he got my phone unlock in 15/20minutes. You don't have to trust me, people on forums also talk about him, thats how I actually learn about him.

Which service did you use? Can you send me the link?

**Comment by Egzotyczny12** [comment]
You need to pay fck xiaomi
